product_docs/docs/pem/9/certificates/index.mdx

```diff
@@ -39,23 +39,23 @@ SSLCertificateFile /path/to/your_domain_name.crt
 SSLCertificateKeyFile /path/to/your_private.key
 ```
 
-To increase security, you can replace the HTTPD self-signed SSL certificates with trusted third-party CA signed certificates in PEM. For more information, see [Replacing HTTPD self-signed SSL certificates](https://www.enterprisedb.com/postgres-tutorials/how-replacing-httpd-self-signed-ssl-certificates-trusted-ca-signed-certificates).
+To increase security, you replace the httpd self-signed SSL certificates with trusted third-party CA signed certificates in PEM. For more information, see [Replacing httpd self-signed SSL certificates](https://www.enterprisedb.com/postgres-tutorials/how-replacing-httpd-self-signed-ssl-certificates-trusted-ca-signed-certificates).
 
 ## PEM backend database server and agent connection certificates
 
 PEM, by default, implements secured SSL/TLS connections between PEM agents and the backend database, and acts as its own certificate authority (CA) to generate certificates and keys for the PEM server and agent. The self-signed SSL certificates and keys for the PEM server and agent are generated during PEM installation. These certificates and keys encrypt the connection from agent to server. In addition, PEM agents are authenticated using their certificate rather than a password.
 
-You can replace the PEM self-signed SSL certificates and keys with the trusted third-party CA certificates. For more information, see [Trusted third-party CA certificates and keys](/pem/latest/certificates/#trusted-third-party-ca-certificates-and-keys).
+You replace the PEM self-signed SSL certificates and keys with the trusted third-party CA certificates and keys. For more information, see [Trusted third-party CA certificates and keys](/pem/latest/certificates/#trusted-third-party-ca-certificates-and-keys).
```

Comment:
I suggest we reinstate 'can' in this sentence to make it clear that this is an optional step.